### PR TITLE
modify configure-gmsa to align with lastest 1.22 version

### PR DESCRIPTION
**Pull request by superleo**

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/configure-gmsa.md [hunk 1]
```diff
@@ -381,6 +381,70 @@ As Pod specs with GMSA fields populated (as described above) are applied in a cl
...
+However, when attempting to browse the directory using the hostname
+-->
+但是，当尝试使用主机名浏览目录时:
+```PowerShell
+cd \\adserver.ad.local\test
+```
+<!--
+you see an error that implies the target share doesn't exist:
+-->
+您会看到一个错误，提示目标共享不存在
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-gmsa.md:420]:
```suggestion
你会看到一个错误，提示目标共享不存在：

```

File: content/zh/docs/tasks/configure-pod-container/configure-gmsa.md [hunk 2]
```diff
@@ -434,32 +496,34 @@ If the DNS and communication test passes, next you will need to check if the Pod
...
 ```
 
 <!--
-If the above command corrects the error, you can automate the step by adding the following lifecycle hook to your Pod spec.  If it did not correct the error, you will need to examine your credspec again and confirm that it is correct and complete.
+If the above corrects the error, you can automate the step by adding the following lifecycle hook to your Pod spec.  If it did not correct the error, you will need to examine your credspec again and confirm that it is correct and complete.
 -->
-如果上述命令修复了错误，你就可以通过向你的 Pod 规约添加生命周期回调来将此操作
-自动化。如果上述命令未能奏效，你就需要再次检查凭据规约，以确保其数据时正确的
-而且是完整的。
+如果以上纠正了错误，您可以通过将以下生命周期钩子添加到您的 Pod 规范来自动执行该步骤。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-gmsa.md:525]:
```suggestion
如果以上命令修复了错误，你可以通过将以下生命周期回调添加到你的 Pod 规约中来自动执行该步骤。
```

**Comment by tengqm** [comment]
Please 1. review the style guide at https://kubernetes.io/zh/docs/contribute/localization_zh/; 2. sign the CLA.

File: content/zh/docs/tasks/configure-pod-container/configure-gmsa.md [hunk 3]
```diff
@@ -434,32 +496,34 @@ If the DNS and communication test passes, next you will need to check if the Pod
...
+如果它没有更正错误，您将需要再次检查您的 credspec 并确认它是正确和完整的。
```
[the remark this answers is not in the document]
Comment (superleo) [reply content/zh/docs/tasks/configure-pod-container/configure-gmsa.md]: ok, fixed

File: content/zh/docs/tasks/configure-pod-container/configure-gmsa.md [hunk 4]
```diff
@@ -381,6 +381,70 @@ As Pod specs with GMSA fields populated (as described above) are applied in a cl
...
+```PowerShell
+ping adserver.ad.local
+```
+<!--
+and correctly resolves the hostname to an IPv4 address. The output is similar to:
+-->
+
+主机名可以被解析为IPV4地址，输出类似如下所示：
+```
+Pinging adserver.ad.local [192.168.111.18] with 32 bytes of data:
```
[the remark this answers is not in the document]
Comment (superleo) [reply content/zh/docs/tasks/configure-pod-container/configure-gmsa.md:408]: ok fixed

File: content/zh/docs/tasks/configure-pod-container/configure-gmsa.md [hunk 5]
```diff
@@ -381,6 +381,70 @@ As Pod specs with GMSA fields populated (as described above) are applied in a cl
...
 
+<!--
+## Containerd 
+
+On Windows Server 2019, in order to use GMSA with containerd, you must be running OS Build 17763.1817 (or later) which can be installed using the patch [KB5000822](https://support.microsoft.com/en-us/topic/march-9-2021-kb5000822-os-build-17763-1817-2eb6197f-e3b1-4f42-ab51-84345e063564). 
+
+There is also a known issue with containerd that occurs when trying to connect to SMB shares from Pods. Once you have configured GMSA, the pod will be unable to connect to the share using the hostname or FQDN, but connecting to the share using an IP address works as expected. 
+-->
+## Containerd
+在Windows Server 2019上对containerd使用GMSA， 需要使用Build 17763.1817 (或更新的版本)，您可以安装 补丁 [KB5000822](https://support.microsoft.com/en-us/topic/march-9-2021-kb5000822-os-build-17763-1817-2eb6197f-e3b1-4f42-ab51-84345e063564).
```
[the remark this answers is not in the document]
Comment (superleo) [reply content/zh/docs/tasks/configure-pod-container/configure-gmsa.md]: ok fixed

File: content/zh/docs/tasks/configure-pod-container/configure-gmsa.md [hunk 6]
```diff
@@ -381,32 +381,107 @@ As Pod specs with GMSA fields populated (as described above) are applied in a cl
...
 
+<!--
+## Containerd 
+
+On Windows Server 2019, in order to use GMSA with containerd, you must be running OS Build 17763.1817 (or later) which can be installed using the patch [KB5000822](https://support.microsoft.com/en-us/topic/march-9-2021-kb5000822-os-build-17763-1817-2eb6197f-e3b1-4f42-ab51-84345e063564). 
+
+There is also a known issue with containerd that occurs when trying to connect to SMB shares from Pods. Once you have configured GMSA, the pod will be unable to connect to the share using the hostname or FQDN, but connecting to the share using an IP address works as expected. 
+-->
+## Containerd
+在Windows Server 2019上对 containerd 使用 GMSA ， 需要使用 Build 17763.1817 (或更新的版本)，
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-gmsa.md:392]:
```suggestion
在 Windows Server 2019 上对 containerd 使用 GMSA，需要使用 Build 17763.1817 (或更新的版本)， 
```

File: content/zh/docs/tasks/configure-pod-container/configure-gmsa.md [hunk 7]
```diff
@@ -381,32 +381,107 @@ As Pod specs with GMSA fields populated (as described above) are applied in a cl
...
+
+On Windows Server 2019, in order to use GMSA with containerd, you must be running OS Build 17763.1817 (or later) which can be installed using the patch [KB5000822](https://support.microsoft.com/en-us/topic/march-9-2021-kb5000822-os-build-17763-1817-2eb6197f-e3b1-4f42-ab51-84345e063564). 
+
+There is also a known issue with containerd that occurs when trying to connect to SMB shares from Pods. Once you have configured GMSA, the pod will be unable to connect to the share using the hostname or FQDN, but connecting to the share using an IP address works as expected. 
+-->
+## Containerd
+在Windows Server 2019上对 containerd 使用 GMSA ， 需要使用 Build 17763.1817 (或更新的版本)， 
+你可以安装补丁 [KB5000822](https://support.microsoft.com/en-us/topic/march-9-2021-kb5000822-os-build-17763-1817-2eb6197f-e3b1-4f42-ab51-84345e063564).
+containerd 场景从 Pod 连接 SMB 共享的时候有一个已知问题： 
+配置了 GMS 以后，无法通过主机名或者 FQDN 访问 SMB共享， 但是通过 IP 地址访问没有问题。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-gmsa.md:395]:
```suggestion
配置了 GMSA 以后，无法通过主机名或者 FQDN 访问 SMB共享，但是通过 IP 地址访问没有问题。
```

File: content/zh/docs/tasks/configure-pod-container/configure-gmsa.md [hunk 8]
```diff
@@ -381,32 +381,107 @@ As Pod specs with GMSA fields populated (as described above) are applied in a cl
...
+
+```PowerShell
+ping adserver.ad.local
+```
+
+<!--
+and correctly resolves the hostname to an IPv4 address. The output is similar to:
+-->
+
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-gmsa.md:404]: delete this line

File: content/zh/docs/tasks/configure-pod-container/configure-gmsa.md [hunk 9]
```diff
@@ -381,32 +381,107 @@ As Pod specs with GMSA fields populated (as described above) are applied in a cl
...
+主机名可以被解析为IPv4地址，输出类似如下所示：
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-gmsa.md:405]:
```suggestion
主机名可以被解析为 IPv4 地址，输出类似如下所示：
```

File: content/zh/docs/tasks/configure-pod-container/configure-gmsa.md [hunk 10]
```diff
@@ -381,32 +381,107 @@ As Pod specs with GMSA fields populated (as described above) are applied in a cl
...
+切换到共享中的目录后，你会看到类似于以下内容的提示：
+
+```
+Microsoft.PowerShell.Core\FileSystem::\\192.168.111.18\test>
+```
+
+<!--
+To correct the behaviour you must run the following on the node `reg add "HKLM\SYSTEM\CurrentControlSet\Services\hns\State" /v EnableCompartmentNamespace /t REG_DWORD /d 1` to add the required registry key. This node change will only take effect in newly created pods, meaning you must now recreate any running pods which require access to SMB shares.
+-->
+要解决问题，你需要在节点上运行以下命令以添加所需的注册表项 `reg add "HKLM\SYSTEM\CurrentControlSet\Services\hns\State" /v EnableCompartmentNamespace /t REG_DWORD /d 1` 。此更改只会在新创建的 pod 中生效，这意味着你必须重新创建任何需要访问 SMB 共享的正在运行的 pod。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-gmsa.md:459]:
```suggestion
要解决问题，你需要在节点上运行以下命令以添加所需的注册表项
`reg add "HKLM\SYSTEM\CurrentControlSet\Services\hns\State" /v EnableCompartmentNamespace /t REG_DWORD /d 1`。
此更改只会在新创建的 Pod 中生效，这意味着你必须重新创建任何需要访问 SMB 共享的正在运行的 Pod。
```

File: content/zh/docs/tasks/configure-pod-container/configure-gmsa.md [hunk 11]
```diff
@@ -434,32 +509,37 @@ If the DNS and communication test passes, next you will need to check if the Pod
...
 
 <!--
-If the above command corrects the error, you can automate the step by adding the following lifecycle hook to your Pod spec.  If it did not correct the error, you will need to examine your credspec again and confirm that it is correct and complete.
+If the above corrects the error, you can automate the step by adding the following lifecycle hook to your Pod spec.  If it did not correct the error, you will need to examine your credspec again and confirm that it is correct and complete.
 -->
-如果上述命令修复了错误，你就可以通过向你的 Pod 规约添加生命周期回调来将此操作
-自动化。如果上述命令未能奏效，你就需要再次检查凭据规约，以确保其数据时正确的
-而且是完整的。
+如果以上命令修复了错误，你可以通过将以下生命周期回调添加到你的 Pod 规约中来自动执行该步骤。
+如果这些操作没有修复错误，你将需要再次检查你的，你将需要再次检查你的 credspec 并确认它是正确和完整的。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/configure-gmsa.md:542]:
```suggestion
如果这些操作没有修复错误，你将需要再次检查你的 credspec 并确认它是正确和完整的。
```

**Review by tengqm — COMMENTED** [review]
pls squash your PR into ONE commit.

**Pull request closed without merging**